app/src/main/java/explorer/city/com/cityexplorer/view/activity/MainActivity.kt

```diff
@@ -18,10 +18,7 @@ import java.util.concurrent.TimeUnit
 
 class MainActivity : LifecycleActivity() {
 
-    companion object {
-        val TAG: String = "MainActivity"
-    }
-
+    val TAG: String = "MainActivity"
```

Comment:
https://kotlinlang.org/docs/reference/java-to-kotlin-interop.html
`companion` is used instead of `static`